doorway-ui-components/src/global/tokens/colors.scss

```diff
@@ -16,8 +16,8 @@
   --bloom-color-green-300: #b4e5be;
   --bloom-color-green-100: #e7f7ea;
 
-  // TODO -- update this with an actual UX color.
-  --doorway-color-blue-1100: #162f4d;
+  --doorway-color-blue-900: #001830;
```

Comment:
I'm not sure how the numbers are differentiated but our doorway blues are waaay darker than the bloom equivalents.

Also to avoid confusion between `doorway-color-blue-800` and `bloom-color-blue-800` lets perhaps call 0F2B50 like doorway-blue-1100 and 001830 doorway-blue-1300? I am sort of guessing at numbers here becuase I don't know how they're calculated but IMO it'd be nice to have higher numbers approx equal darker regardless of whether its `bloom` or `doorway`
![image](https://github.com/metrotranscom/doorway/assets/10789523/5a7010ea-f964-4692-8b0d-794375f3e2f8)

Reply:
Done, renamed doorway-color-blue-800 to 1100 and doorway-color-blue-900 to 1300 and asked in the UX chat if the corresponding colors in Figma can also be renamed.